sources/lib/datalab/datalab/notebook/static/visualization.ts

```diff
@@ -58,22 +65,22 @@ define("visualization", function() {
     }
   });
 
-  return {
-    load: function load(name, req, callback, config) {
-      if (config.isBuild) {
-        loadCallback(null);
+  export function load(name: any, req: any, callback: any, config: any) {
+    if (config.isBuild) {
+      loadCallback(null);
```

Comment:
Good question. That code is over a year old, and that code path probably never gets executed. The callback argument should be renamed and line 20 removed.